### PR TITLE
document principles

### PR DESCRIPTION
**Pull request by BrianHicks**

### CODE REVIEW

File: docs/principles.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Principles
+
+These principles inform how we make decisions about rbt, both in how it should feel to use and as standards for internals:
+
+1. **Be Transparent**
+
+   As an rbt user, you should be able to observe what rbt is doing at any given time: it should never block or hang without any kind of feedback.
+   You to be able to understand what rbt will do once you've written down your build instructions.
```
Comment (bhansconnect) [inline docs/principles.md:8]: This sentence is worded oddly. "You to be able to understand"

File: docs/principles.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+   We work to minimize hidden or undocumented variables that can come into play to break your build, and try to make sure that rbt supports you in these situations.
+
+2. **Be Approachable**
+
+   When we say that rbt should be approachable, we're thinking about teams producing software.
+   Say the build needs some adjustments: do you need to be an expert in the build system, or can you just go look at the code, make untuitive changes, try them out, and get a good result?
```
Comment (bhansconnect) [inline docs/principles.md:15]: untuitive -> intuitive

File: docs/principles.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+   We don't shy away from complexity where it's warranted, but we think that straightforward and readable code is valuable and will help keep the system working well.
+   (After all, "a complex system that works is invariably found to have evolved from a simple system that worked.")
+
+3. **Be Fast**
+
+   This has one plain meaning: rbt should run a build to completion as quickly as possible.
+
+   However, since build systems are typically bound by the speed of the tools you're using to build, rbt also focuses on perceived speed: we show you feedback as soon as possible and work hard to minimize startup costs.
+
+These asre listed in order of importance.
```
Comment (bhansconnect) [inline docs/principles.md:28]: asre -> are

File: docs/principles.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+   When it comes to internal code, approachability means we try to keep internals simple and well-documented.
+   We don't shy away from complexity where it's warranted, but we think that straightforward and readable code is valuable and will help keep the system working well.
+   (After all, "a complex system that works is invariably found to have evolved from a simple system that worked.")
+
+3. **Be Fast**
+
+   This has one plain meaning: rbt should run a build to completion as quickly as possible.
+
+   However, since build systems are typically bound by the speed of the tools you're using to build, rbt also focuses on perceived speed: we show you feedback as soon as possible and work hard to minimize startup costs.
+
```
Comment (bhansconnect) [inline docs/principles.md:27]: Add some sort of separation between 3 and the following paragraph?

Comment (bhansconnect) [reply docs/principles.md:27]: Maybe just a `-----`

**Review by bhansconnect — APPROVED** [review]

**Comment by BrianHicks** [comment]
guess you can tell I don't have a spellchecker turned on in my editor, huh?

I've installed one in CI 😆

**Comment by BrianHicks** [comment]
ahhh, boo. So much for the easy solution.

**Pull request merged**